aws_lambda_powertools/utilities/parameters/appconfig.py

```diff
@@ -91,6 +88,9 @@ def __init__(
         self.environment = environment
         self.current_version = ""
 
+        self.next_call = ""
```

Comment:
```suggestion
        self._next_token = ""
```

Unless we're planning to expose the next token to customers, let's make it private.